src/vulkan-renderer/tools/file.cpp

```diff
@@ -2,44 +2,56 @@
 
 #include <spdlog/spdlog.h>
 
+#include <algorithm>
 #include <cassert>
 #include <fstream>
 
 namespace inexor::vulkan_renderer::tools {
 
-bool File::load_file(const std::string &file_name) {
+bool does_file_exist(const std::string &file_name) {
     assert(!file_name.empty());
 
-    // Open stream at the end of the file to read it's size.
-    std::ifstream file_to_load(file_name.c_str(), std::ios::in | std::ios::binary | std::ios::ate);
-
-    if (file_to_load.is_open()) {
-        spdlog::debug("File {} has been opened.", file_name);
+    std::ifstream file_to_load(file_name.c_str(), std::ios::in);
+    return file_to_load.is_open();
```

Comment:
```suggestion
    return std::filesystem::exists(file_name);
```

Reply:
Oh nice. In this case we don't need this wrapper around it at all. Let me remove that.

Reply:
You are right, I will remove it. It's not used currently but I want to use it in the glTF2 loader code.